AirCasting/Graph/GraphView.swift

```diff
@@ -40,6 +44,17 @@ struct GraphView: View {
 
 struct GraphView_Previews: PreviewProvider {
```

Comment:
Please wrap in `#if DEBUG`

Reply:
Well, I'm a little confused. I agree with you, that wrapping preview in `#if DEBUG` is a good idea. But we have lots of previews not wrapped in DEBUG, shouldn't we be consistent? 
I can wrap this one, but in my opinion either we should add a ticket to Trello to wrap them all, or I need an explanation why should I wrap this one and others can be unwrapped.

Reply:
I added  `#if DEBUG` here, but I think that we should refactor the code so that all previews will be wrapped i  `#if DEBUG` :)